validator/parse-css.js

```diff
@@ -827,104 +350,341 @@ Canonicalizer.prototype.blockTypeFor = function(atRule) {
 };
 
 /**
- * @param {!parse_css.Stylesheet} stylesheet
+ * Parses and returns a list of rules, such as at the top level of a stylesheet.
+ * Return list has only QualifiedRule's and AtRule's as top level elements.
+ * @param {!Array<!parse_css.CSSParserToken>} tokenList
+ * @param {boolean} topLevel
  * @param {!Array<!parse_css.ErrorToken>} errors output array for the errors.
- * @return {!parse_css.Stylesheet}
+ * @return {!Array<!parse_css.CSSParserRule>}
  */
-Canonicalizer.prototype.canonicalizeStylesheet = function(stylesheet, errors) {
-  for (let i = 0; i < stylesheet.value.length; i++) {
-    const currentRule = stylesheet.value[i];
-    if (currentRule instanceof parse_css.EOFToken) {
-      stylesheet.eof = currentRule;
-    } else if (currentRule instanceof parse_css.QualifiedRule) {
-      stylesheet.rules.push(
-          this.canonicalizeQualifiedRule(currentRule, errors));
-    } else if (currentRule instanceof parse_css.AtRule) {
-      stylesheet.rules.push(this.canonicalizeAtRule(currentRule, errors));
+Canonicalizer.prototype.parseAListOfRules = function(
```

Comment:
Typically in our styleguide we tend to avoid articles "a" and "the". Is there any reason we are using them here?